apps/emqx_bridge_tablestore/src/emqx_bridge_tablestore.erl

```diff
@@ -0,0 +1,315 @@
+                )
+            }
+        }
+    ].
+
+connector_values(timeseries) ->
+    #{
+        name => <<"tablestore_connector">>,
+        enable => true,
+        endpoint => <<"https://myinstance.cn-hangzhou.ots.aliyuncs.com">>,
+        storage_model_type => timeseries,
+        instance_name => <<"myinstance">>,
+        access_key_id => <<"******">>,
+        access_key_secret => <<"******">>
+    }.
+
+action_parameters_example(timeseries) ->
+    #{
+        parameters => #{
+            storage_model_type => timeseries,
+            table_name => <<"table_name">>,
+            data_source => <<"${data_source}">>,
+            measurement => <<"${measurement}">>,
+            tags => #{
+                tag1 => <<"${tag1}">>,
+                tag2 => <<"${tag2}">>
+            },
+            fields => [
+                #{
+                    column => <<"${column}">>,
+                    value => <<"${value}">>,
+                    isint => true
+                }
+            ],
+            meta_update_model => <<"MUM_IGNORE">>
+        }
+    }.
+
+values(StorageType, get) ->
+    values(StorageType, post);
+values(StorageType, put) ->
+    values(StorageType, post);
+values(timeseries, post) ->
+    #{
+        name => <<"tablestore_connector">>,
+        enable => true,
+        local_topic => <<"local/topic/#">>,
+        endpoint => <<"https://myinstance.cn-hangzhou.ots.aliyuncs.com">>,
+        storage_model_type => timeseries,
+        instance_name => <<"myinstance">>,
+        access_key_id => <<"******">>,
+        access_key_secret => <<"******">>,
+        table_name => <<"table_name">>,
+        measurement => <<"measurement">>,
+        tags => #{
+            tag1 => <<"${tag1}">>,
+            tag2 => <<"${tag2}">>
+        },
+        fields => [
+            #{
+                column => <<"${column}">>,
+                value => <<"${value}">>,
+                isint => true
+            }
+        ],
+        data_source => <<"${data_source}">>,
+        meta_update_model => <<"MUM_IGNORE">>,
+        resource_opts => #{
+            batch_size => 1,
+            batch_time => <<"20ms">>
+        },
+        pool_size => 8,
+        ssl => #{enable => false}
+    }.
+
+%% -------------------------------------------------------------------------------------------------
+%% Hocon Schema Definitions
+namespace() -> "bridge_tablestore".
+
+roots() -> [].
+
+fields("connector") ->
+    [
+        storage_model_type_field(),
+        {endpoint, mk(binary(), #{required => true, desc => ?DESC("desc_endpoint")})},
+        {instance_name, mk(binary(), #{required => true, desc => ?DESC("desc_instance_name")})},
+        {access_key_id,
+            emqx_schema_secret:mk(
+                #{
+                    required => true,
+                    desc => ?DESC("desc_access_key_id")
+                }
+            )},
+        {access_key_secret,
```

Comment:
if you name it `secret_access_key`, then the redact util already have it covered. otherwise add it to the redact key-words list.